dwst/scripts/lib/particles.js

```diff
@@ -120,18 +127,22 @@ function readInstructionName(parsee) {
     throw new InvalidParticles('an instruction name', String(parsee));
   }
   if (parsee.startsWith('}') || parsee.length === 0) {
-    throw new InvalidParticles(['('], String(parsee));
+    throw new InvalidParticles(quoteEach('('), String(parsee));
   }
   return instructionName;
 }
 
 function readInstructionArg(parsee) {
   const arg = parsee.readWhile(alphaChars.concat(digitChars));
-  if (parsee.startsWith('}') || arg.length === 0) {
+  if (parsee.startsWith('}')) {
```

Comment:
I don't understand this. We just read an argument. Checking for `}` specifically seems like a completely random thing to do.

Reply:
I agree. I think we inherited checking '}' here from the time when the function used readUntil and stoppers like ',' and ')'.

Reply:
Oh. I didn't realise it was present in the old code.